schemas/logs/fluentd/windows_eventlog.yml

```diff
@@ -0,0 +1,136 @@
+schema: Fluentd.WindowsEventlog
+parser:
+  native:
+    name: Fluentd.WindowsEventlog
+description: Fluentd Windows Event Log v1 Plugin Parser
+referenceURL: https://github.com/fluent/fluent-plugin-windows-eventlog
+version: 0
+fields:
+- name: channel
+  required: true
+  description: Identifies the type of log (i.e Application, Security, System).
+  type: string
+- name: computer_name
+  required: true
+  description: The name of the machine that logged the event. 
+  type: string
+- name: description_title
+  required: false
```

Comment:
You can remove any `required: false` since it is the default. It is going to saves few lines from the YAML file

Reply:
Removed the required: false fields

Reply:
I will circle back when the cli tool comes out.